src/shared/components/Nexus/Projects.tsx

```diff
@@ -0,0 +1,38 @@
+import { useNexus } from '@bbp/react-nexus';
+import { ProjectList, ProjectListOptions, Project } from '@bbp/nexus-sdk';
+
+const Get = ({
+  orgLabel,
+  projectLabel,
+  children,
+}: {
+  orgLabel: string;
+  projectLabel: string;
+  children: any;
+}) => {
+  const state = useNexus<Project>(nexus =>
+    nexus.Project.get(orgLabel, projectLabel)
+  );
```

Comment:
You probably want to to update on orgLabel/projectLabel change? what about options? like deprecated?